dev/com.ibm.ws.io.opentracing.opentracing-api.0.30.0/build.xml

```diff
@@ -0,0 +1,22 @@
+<!--
```

Comment:
don't need a build.xml here, it won't do anything.

Reply:
Ok, i'll remove - got this from some other proj